sacro/views.py

```diff
@@ -199,6 +201,28 @@ def review_detail(request, pk):
         "approved_outputs_url": approved_outputs_url,
         "counts": counts,
         "review": review,
+        "summary_url": summary_url,
     }
 
     return TemplateResponse(request, "review.html", context=context)
+
+
+@require_POST
+def summary(request, pk):
```

Comment:
Why is this a POST rather than a GET?

Reply:
IIRC we have to POST to get a file save dialogue without the page changing when someone clicks the download button.  I mirrored the approved_outputs view for this.